atomic_reactor/schemas/user_params.json

```diff
@@ -31,6 +31,26 @@
     "koji_task_id": {"type": "integer"},
     "koji_upload_dir": {"type": "string"},
     "name": {"type": "string"},
+    "parent_images_digests": {
+      "type": "object",
+      "patternProperties": {
+        "^.+:.+": {
+          "description": "Image with tags",
+          "type": "object",
+          "patternProperties": {
+            "^.+": {
+              "description": "Platform specific digests",
+              "type": "string",
+              "pattern": "@sha256:"
```

Comment:
Technically 'sha256' is the algorithm name, which will change in future.

Reply:
replaced with generic pattern